### DISCUSSION TITLE
Listen on random port

### DISCUSSION THREAD

**Discussion by zdm** [uNetworking/uWebSockets.js#729] [Q&A] [answered]
Hi,
Is it possible to listen on random port and get this port in listen callback somehow?

**Comment by ghost** [comment, marked as answer]
https://unetworking.github.io/uWebSockets.js/generated/index.html#us_socket_local_port

I think you pass 0 as port, then get it

**Comment by zdm** [comment]
Yes, this works.
Thank you.
